shc/web/interface.py

```diff
@@ -81,14 +102,14 @@ def __init__(self, host: str, port: int, index_name: Optional[str] = None, root_
         self._associated_tasks: weakref.WeakSet[asyncio.Task] = weakref.WeakSet()
         # last will (object, value) per API websocket client (if set)
         self._api_ws_last_will: Dict[aiohttp.web.WebSocketResponse, LastWillT] = {}
-        # data structure of the user interface's main menu
+        # using class `MenuEntrySpec` and `SubMenuEntrySpec` as data structure for the user interface's main menu
         # The structure looks as follows:
-        # [('Label', 'icon', 'page_name'),
-        #  ('Submenu label', None, [
-        #     ('Label 2', 'icon', 'page_name2'), ...
+        # [('Label', 'icon', 'page_name', 'is_active'),
+        #  ('Submenu label', 'icon', None, 'is_active' [
+        #     ('Label 2', 'icon', 'page_name2', 'is_active'), ...
```

Comment:
This comment should be adjusted for the new classes, which replace the old tuple-based structure. I.f.

```python
        # The structure looks as follows:
        # [MenuEntrySpec('Label', 'icon', 'page_name', false),
        #  SubMenuEntrySpec('Submenu label', 'icon', None, true, [
        #     MenuEntrySpec('Label 2', 'icon', 'page_name2', true), ...
```

Reply:
you're so right. Fixed.